data/projects/cobot_enhancement.md

```diff
@@ -0,0 +1,113 @@
+worthwile having good up-to-date docs. We aim at using topic modeling i.e. take
+help of machine learning and natural language processing(NLP) for this one.
+
+Repository: https://github.com/coala/corobo
+Proposal PDF: https://meetmangukiya.github.io/GSoC_cobot_enhancement.pdf
+
+#### Milestones
+
+##### PREPARATION/BONDING
+
+Milestone: https://gitlab.com/coala/GSoC-2017/milestones/2
+
+* The applicant has made a cEP of how the new plugins will be implemented, how
+  will it be tested, etc.
+
+
+##### CODING PHASE 1
+
+Milestone: https://gitlab.com/coala/GSoC-2017/milestones/23
+
+* Port all the plugins to errbot.
+* Write half of the new features proposed.
+* Write the test helper and all the merged plugins should have unit tests.
+
+##### CODING PHASE 2
+
+Milestone: https://gitlab.com/coala/GSoC-2017/milestones/24
+
+* Write the other half of the new features proposed.
+* Start implementing the topic modeling functions, etc.
+
+##### CODING PHASE 3
+
+Milestone: https://gitlab.com/coala/GSoC-2017/milestones/25
+
+* Topic modeling should be implemented completely and should also be able to
+  trace back to the related document in the training data.
+* Document all the bot commands and the topic modeling process used, for future
+  references.
+
+#### Process
+
+##### CODING PHASE 1
+
+In week 1 and 2: All the existing scripts in gitlab/coala/cobot should be
+ported to github/coala/corobo. All the plugins should be tested with 100%
+coverage.
+
+In week 3 and 4: At least two new plugins from
+https://github.com/coala/corobo/milestone/5 should be written and tested.
+
+##### CODING PHASE 2
+
+In week 5 and 6:
+1. Get all the PRs from CODING PHASE 1 merged.
+2. Write the WSDL file for coala webservices. Generate coala-api-wrapper from
+   the WSDL file and publish to PyPI.
+3. Write all the plugins from https://github.com/coala/corobo/milestone/5 and
+   use coala-api-wrapper instead of making raw API calls.
+
+In week 7 and 8:
+1. Merge all the PRs from week 5 and 6.
+2. Review all the plugin commands and fix any bugs that were detected during
+   this period. Do a UX survey from the coala community about the commands.
+   Do the necessary changes from the insights of the UX survey.
+3. Lay the foundation of topic-modeling and create sane data sets from coala
+   documentation.
+
+##### CODING PHASE 3
+
+In week 9 and 10:
+1. Have least functionality that does the pre-processing and can retrieve topic
+   from the given question, i.e. figuring out what is the expected answer.
+2. Start working on the passage retrieval part to _get_ the answers from the
+   documentation.
+3. Working functionality to get answers from the docs and summarize the doc to
+   answer in concise manner.
+
+In week 11 and 12:
+1. Based on the scores determine if the question can be answered from our docs,
+   if not just return a google result for the same question.
+2. Buffer.
+3. Documentation(particularly about the algorithm used for retrieving the
+   answers).
+
```

Comment:
That is your problem, with your editor / configuration.
You should run linters and read `git show` before submitting to ensure you've fix it.